apps/web/src/app/(home)/hero.tsx

```diff
@@ -2,69 +2,66 @@
 import { QuickLink } from "@/components/quick-link";
 import { Marquee } from "@/components/ui/marquee";
 import Image from "next/image";
-import { useMemo } from "react";
 
-export default function LandingSection(): JSX.Element {
-  // Create array of all image numbers (1-91) in a deterministic order
-  const { firstRowImages, secondRowImages } = useMemo(() => {
-    // Use a simple deterministic pattern instead of random shuffle
-    const allImageNumbers = Array.from({ length: 91 }, (_, i) => i + 1);
-
-    // Create a deterministic but varied order by interleaving
-    const shuffledImages: number[] = [];
-    const step = 7; // Prime number for better distribution
-    let current = 0;
-
-    for (let i = 0; i < 91; i++) {
-      const imageNum = allImageNumbers[current];
-      if (imageNum !== undefined) {
-        shuffledImages.push(imageNum);
-      }
-      current = (current + step) % 91;
-    }
+interface HeroProps {
+  images: string[];
+}
 
-    return {
-      firstRowImages: shuffledImages.slice(0, 45), // Increased from 30 to 45
-      secondRowImages: shuffledImages.slice(45, 91), // Remaining images
-    };
-  }, []);
+export default function LandingSection({ images }: HeroProps): JSX.Element {
+  // Split images into two rows
+  const firstRowImages = images.slice(0, Math.ceil(images.length / 2));
```

Comment:
randomize the images here first! so that on every reload, we will see new images at top